packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -461,14 +470,16 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
         //
         // Explicit checks:
         // - The game must not have already been initialized.
-        // - An output root cannot be proposed at or before the genesis block.
+        // - An output root that is more than 1 day old cannot be proposed.
 
         // INVARIANT: The game must not have already been initialized.
         if (initialized) revert AlreadyInitialized();
 
-        // Do not allow the game to be initialized if the root claim corresponds to a block at or before the
-        // configured genesis block number.
-        if (l2BlockNumber() <= GENESIS_BLOCK_NUMBER) revert UnexpectedRootClaim(rootClaim());
+        // Do not allow the game to be initialized if the root claim corresponds to an L2 block that is more than
+        // 1 day old. Note that this check prevents output roots from being disputed until a chain has been live for
+        // at least 1 day.
```

Comment:
I don't think we should prevent publishing output roots for the first day of a chain (it will make the local devnet unusable for example).

Reply:
Yeah, fair. I can add a check for if this'll underflow.